### PR TITLE
bazel_6: 6.0.0 -> 6.1.0

### PR DESCRIPTION
**Pull request by dmivankov**
Updating the updater script:
- `//external:all` -> `//external:*` to make it work with recent bazel versions

Avoiding chicken-egg problem by explicitly asking to first build previous version, then update sources version, then use updater and previous version to update src-deps.

`bazel_self` may not necessarily build out of the box after just the sources version bump and may depend on updater being run first, which can't run without `bazel_self`. So listing steps to use old version to help updating to new version

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dmivankov** [comment]
@ofborg build python3Packages.tensorflow-probability

File: pkgs/development/python-modules/tensorflow-probability/default.nix [hunk 1]
```diff
@@ -54,7 +54,7 @@ let
     LIBTOOL = lib.optionalString stdenv.isDarwin "${cctools}/bin/libtool";
 
     fetchAttrs = {
-      sha256 = "sha256-pST4R45mWC5j0ngkkRe+hmostaMploW0+BN3WKPt0t0=";
+      sha256 = lib.fakeHash;
```
Comment (dmivankov) [inline pkgs/development/python-modules/tensorflow-probability/default.nix:57]:
```suggestion
      sha256 = "sha256-9i0ExaIeNz7+ddNAoU2ak8JY7lI2aY6eBDiPzJYuJUk=";
```

**Comment by dmivankov** [comment]
https://github.com/ofborg build python3Packages.tensorflow-probability

**Review by uri-canva — APPROVED** [review]

**Comment by wegank** [comment]
@ofborg build bazel_6 bazel_6.passthru.tests

**Pull request merged**